### PR TITLE
docs: Add badges

### PR DESCRIPTION
**Pull request by kaushal07wick**
adding some badges
added a gif to readme for the vectordb repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +75,12 @@ table = db.create_table("my_table",
 result = table.search([100, 100]).limit(2).to_pandas()
 ```
 
+## Let's jump to cooking some cool applications using LanceDB.
```
Comment (AyushExel) [inline README.md:78]: Probably remove this section. We'll need a better copy for this.

File: README.md [hunk 2]
```diff
@@ -5,10 +5,11 @@
 
 **Developer-friendly, serverless vector database for AI applications**
 
-<a href="https://lancedb.github.io/lancedb/">Documentation</a> •
-<a href="https://blog.lancedb.com/">Blog</a> •
-<a href="https://discord.gg/zMM32dvNtd">Discord</a> •
-<a href="https://twitter.com/lancedb">Twitter</a>
+<a href='https://github.com/lancedb/vectordb-recipes/tree/main' target="_blank"><img alt='LanceDB' src='https://img.shields.io/badge/VectorDB_Recipes-100000?style=for-the-badge&logo=LanceDB&logoColor=white&labelColor=645cfb&color=645cfb'/></a>
```
Comment (AyushExel) [inline README.md:8]: Let's keep the gif for now. Just add badges

File: README.md (hunk 1 above)
Comment (kaushal07wick) [reply README.md:78]: ok

File: README.md (hunk 2 above)
Comment (kaushal07wick) [reply README.md:8]: done.

File: vectordb.gif
Comment (AyushExel) [inline vectordb.gif:1]: Remove this gif

**Review by AyushExel — APPROVED** [review]

**Pull request merged**